crates/typst/src/layout/repeat.rs

```diff
@@ -35,6 +36,17 @@ pub struct RepeatElem {
     /// The content to repeat.
     #[required]
     pub body: Content,
+
+    /// The gap between each instance of the body.
+    #[default]
+    pub gap: Length,
+
+    /// Whether to increase the gap between instances to completely fill the
+    /// available space.
+    ///
+    /// When `{auto}`, inherits the paragraph's [`justify`]($par.justify).
+    #[default(Smart::Custom(true))]
+    pub justify: Smart<bool>,
```

Comment:
I'm not sure whether inheriting from `par.justify` is really good. I think `repeat` is typically used to produce a very specific layout and it changing up randomly based on whether the user uses justification could produce unintended results.